src/server/cluster/cluster_family.cc

```diff
@@ -910,6 +900,38 @@ void ClusterFamily::DflyMigrateFullSyncCut(CmdArgList args, ConnectionContext* c
   cntx->SendOk();
 }
 
+void ClusterFamily::DflyMigrateAck(CmdArgList args, ConnectionContext* cntx) {
+  CmdArgParser parser{args};
+  auto sync_id = parser.Next<uint32_t>();
+
+  if (auto err = parser.Error(); err) {
+    return cntx->SendError(err->MakeReply());
+  }
+
+  auto migration = GetOutgoingMigration(sync_id);
+  if (!migration)
+    return cntx->SendError(kIdNotFound);
+
+  if (migration->GetState() != MigrationState::C_FINISHED) {
+    return cntx->SendError("Migration process is not in C_FINISHED state");
+  }
+
+  // TODO implement blocking on migrated slots only
+  const auto deleted_slots = ToSlotSet(migration->GetSlotRange());
+
+  // TODO it's a bug need to rewrite with mutex or fix tl_cluster_config because it's not thread
+  // local
+  tl_cluster_config->RemoveSlots(deleted_slots);
```

Comment:
Can we create a copy of the config, remove slots from it, and then assign the new config in the `Await()` below?

Reply:
Yes it will be done in the next PR or if you want I can add it here

Reply:
This should be done with carefully as a new config might be assinged while you fetch the old one and update it. Note that right now you only remove the slots and do not add them in the config to the target node, you must fix this inorder to get cluster test pass.

Reply:
actually I see the test_cluster_fuzzymigration is green but I dont really understand how.. worth checking

Reply:
we have data race in config applying I will update test tommorow